Online-Messaging-App/src/app/home/createChannel/create-channel.component.scss

```diff
@@ -0,0 +1,24 @@
+.center{
```

Comment:
![image](https://user-images.githubusercontent.com/35238045/73327574-61235380-420b-11ea-91b4-86f3d666dbee.png)
How this looks on a bigger monitor btw. If possible please fix

Reply:
Same issue as above, changed some code up but can't recreate issue